robot/resources/lib/python_keywords/http_gate.py

```diff
@@ -159,24 +161,61 @@ def upload_via_http_gate(cid: str, path: str, endpoint: str, headers: dict = Non
     return resp.json().get("object_id")
 
 
+@allure.step("Check is the passed object complex")
+def is_object_complex(filepath: str) -> bool:
+    """
+    This function check passed file size and return True if file_size > SIMPLE_OBJECT_SIZE
+    filepath: File path to check
+    """
+    file_size = os.path.getsize(filepath)
+    logger.info(f"Size= {file_size}")
+    if file_size > int(SIMPLE_OBJECT_SIZE):
```

Comment:
That is not 100% true. 
Object is complex if it greater than maxObjectSize value in network